.azure/pipelines/ci.yml

```diff
@@ -706,6 +706,7 @@ extends:
       # Source build
       - template: /eng/common/templates-official/job/source-build.yml@self
         parameters:
+          enableInternalSources: true
```

Comment:
I see that this works, but what exactly does it do?

Reply:
It calls the two templates that I commented out.

Reply:
Do we know why it works when doing it this way, but not when calling the templates directly?